src/Compilers/CSharp/Portable/Symbols/Source/SourceMemberContainerSymbol.cs

```diff
@@ -3140,6 +3143,94 @@ void addCloneMethod()
                 members.Add(new SynthesizedRecordClone(this, memberOffset: members.Count, diagnostics));
             }
 
+            MethodSymbol addPrintMembersMethod()
```

Comment:
>addPrintMembersMethod [](start = 25, length = 21)

:memo: this is similar to `addEqualityContract`. In case of a user-defined method, we check accessibility, return type, overrides method from base type, and method is overridable (unless sealed type).